app/views/uploadSupportingMaterialMultiple.scala.html

```diff
@@ -83,6 +83,11 @@
             document
                 .getElementById("file")
                 .addEventListener("change", onFileSelected(id, targetURL, csrfToken));
+
+            window.addEventListener("pageshow", function(event) {
+                const files = document.querySelectorAll('input[type="file"]');
+                files.forEach(input => input.value = '');
+            });
```

Comment:
We shouldn't be adding nested javascript like this. If needed keep it in a separate js file. However, we are encouraged to not use javascript at all on our services. Can we look into an alternative solution?